src/libraries/System.Net.Connections/ref/System.Net.Connections.cs

```diff
@@ -75,7 +75,9 @@ public partial class SocketsConnectionFactory : System.Net.Connections.Connectio
     {
         public SocketsConnectionFactory(System.Net.Sockets.AddressFamily addressFamily, System.Net.Sockets.SocketType socketType, System.Net.Sockets.ProtocolType protocolType) { }
         public SocketsConnectionFactory(System.Net.Sockets.SocketType socketType, System.Net.Sockets.ProtocolType protocolType) { }
+        [System.Runtime.Versioning.UnsupportedOSPlatform("browser")]
```

Comment:
Should the whole class be marked unsupported on browser? Not sure why you’d want to create a sockets connection factory on browser when sockets aren’t supported.